test/Integration/IntegrationTestFixture.cs

```diff
@@ -41,6 +45,44 @@ protected void StartAzurite()
             this.AzuriteHost.Start();
         }
 
+        /// <summary>
+        /// Build the samples-java and test-java projects.
+        /// </summary>
+        private static void BuildJavaFunctionApps()
+        {
+            string samplesJavaPath = Path.Combine(TestUtils.GetPathToBin(), "SqlExtensionSamples", "Java");
+            BuildJavaFunctionApp(samplesJavaPath);
+
+            string testJavaPath = Path.Combine(TestUtils.GetPathToBin(), "..", "..", "..", "Integration", "test-java");
+            BuildJavaFunctionApp(testJavaPath);
+        }
+
+        /// <summary>
+        /// Run `mvn clean package` to build the Java function app.
+        /// </summary>
+        private static void BuildJavaFunctionApp(string workingDirectory)
+        {
+            var maven = new Process()
```

Comment:
And actually do we need this if the pipelines are already doing the build? If we want to have a way for people to easily build latest + run tests locally then generally it'd be good to do that in a script instead (since building can take a while and often times isn't actually needed)